aws_doc_sdk_examples_tools/config/services.yaml

```diff
@@ -1260,15 +1260,15 @@ entityresolution:
   sort: AWS Entity Resolution
```

Comment:
The sort key must be the name without 'Amazon' or 'AWS' at the front.

Reply:
Addressed this comment.